pkg/drivers/libhvee/libhvee_windows.go

```diff
@@ -0,0 +1,309 @@
+		PrepareSystemSettings(d.MachineName, nil).
+		PrepareMemorySettings(
+			func(ms *hypervctl.MemorySettings) {
+				ms.DynamicMemoryEnabled = d.DynamicMemory
+				ms.VirtualQuantity = uint64(d.Memory)
+				ms.Reservation = 1024
+				ms.Limit = uint64(d.Memory)
+			}).
+		PrepareProcessorSettings(
+			func(ps *hypervctl.ProcessorSettings) {
+				ps.VirtualQuantity = uint64(d.CPU)
+			}).
+		Build()
+
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> creating: copy disk image")
+	diskPath := d.getDiskPath()
+	if err := crcos.CopyFile(d.ImageSourcePath, diskPath); err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> creating: hardware setup")
+	err = hypervctl.NewDriveSettingsBuilder(systemSettings).
+		AddScsiController().
+		AddSyntheticDiskDrive(0).
+		DefineVirtualHardDisk(diskPath,
+			func(vhdss *hypervctl.VirtualHardDiskStorageSettings) {}).
+		Finish().
+		Finish().
+		Finish().
+		Complete()
+
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> creating: done")
+
+	return nil
+}
+
+func (d *Driver) getMachine() (*hypervctl.VirtualMachine, error) {
+	vmm := hypervctl.NewVirtualMachineManager()
+
+	log.Debugf("Machine: libhvee -> get machine")
+	return vmm.GetMachine(d.MachineName)
+}
+
+// Start starts an host
+func (d *Driver) Start() error {
+	vm, err := d.getMachine()
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> start")
+	return vm.Start()
+}
+
+// Stop stops an host
+func (d *Driver) Stop() error {
+	vm, err := d.getMachine()
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> stop")
+	return vm.Stop()
+}
+
+// Remove removes an host
+func (d *Driver) Remove() error {
+	s, err := d.GetState()
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> remove running")
+	if s == state.Running {
+		if err := d.Kill(); err != nil {
+			return err
+		}
+	}
+
+	vm, err := d.getMachine()
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> remove vm")
+	err = vm.Remove("")
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> remove disk file")
+	return os.Remove(d.getDiskPath())
+}
+
+// Kill force stops an host
+func (d *Driver) Kill() error {
+	vm, err := d.getMachine()
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Machine: libhvee -> stop with force")
+	return vm.StopWithForce()
+}
+
+func (d *Driver) GetIP() (string, error) {
+	return "", fmt.Errorf("")
+}
+
+func (d *Driver) GetSharedDirs() ([]drivers.SharedDir, error) {
+	for _, dir := range d.SharedDirs {
+		if !smbShareExists(dir.Tag) {
+			return []drivers.SharedDir{}, nil
+		}
+	}
+	return d.SharedDirs, nil
+}
+
+func (d *Driver) getDiskPath() string {
+	return d.ResolveStorePath(fmt.Sprintf("%s.%s", d.MachineName, d.ImageFormat))
+}
+
+func (d *Driver) resizeDisk(newSize int64) error {
+	diskPath := d.getDiskPath()
+	out, err := cmdOut(fmt.Sprintf("@(Get-VHD -Path %s).Size", quote(diskPath)))
```

Comment:
No API in libhvee for this and `Hyper-V\\Resize-VHD`?

Reply:
No. Does not exist. Suggestion for improvement?

Reply:
Maybe you can add a link to the issue as a comment

Reply:
Brent proposed something and is being reviewed.